app/controllers/pet_applications_controller.rb

```diff
@@ -5,11 +5,19 @@ def index
   end
 
   def update
-    pet = Pet.find(params[:pet_id])
     applicant = Applicant.find(params[:applicant])
     params[:status] = "pending"
+    if params[:approve_all]
```

Comment:
This logic works but we should review it. I feel like it will break something in the future.